cmd/kubeadm/app/cmd/init.go

```diff
@@ -31,15 +31,16 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/validation"
+	cmdphases "k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases"
```

Comment:
I've brought this up in the past and haven't really seen anything change. I don't see why we use package aliases when the package name should either be descriptive enough or using the alias makes it longer (See https://blog.golang.org/package-names).

Especially with the packages we control under `k8s.io/kubernetes/cmd/kubeadm`, I really think we should avoid this problem. This also causes the same package to be called different things in different files which makes readability across files difficult.